packages/docs/src/stories/templates/carousel.stories.ts

```diff
@@ -45,7 +45,7 @@ export const CarouselWithImages = {
 export const LoopAndAutoplay = {
   name: 'Loop and Autoplay',
   render: () => html`
-    <sd-carousel loop autoplay>
+    <sd-carousel loop autoplay role="region" aria-label="Carousel in loop and autoplay">
```

Comment:
Question on that: Shouldn't this be part of the component itself? Would this work?

Reply:
Nevermind. I just saq, that this is the whole idea of the PR. :D